### DISCUSSION TITLE
Plotting efficiency Windows vs Linux

### DISCUSSION THREAD

**Discussion by zrjaa1** [Chia-Network/chia-blockchain#2081] [General]
Hi guys, I'm new to Chia and have done some plotting in Windows. I noticed that people are plotting on different platforms and wondering if there is any comparison between the efficiency of plotting on different platforms with the same hardware? For example, how does the efficiency in Windows compare with that in Linux? 

If there is a comparison with more platforms involved that would be even better. Any idea? Thanks!

**Comment by cccat6** [comment]
See [here](https://www.chia.net/2021/02/22/plotting-basics.html)
It says "It is worth noting that Windows suffers 5-10% slower plot times versus MacOS or Linux for now."

**Reply by zrjaa1** [reply]
This is exactly what I'm looking for. Thanks.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
